website/docs/guides/migration/versions/02-upgrading-to-v1.5.md

```diff
@@ -53,9 +56,9 @@ More to come!
 :::
 
 ### Publishing models as APIs
```

Comment:
Some of the nomenclature here is liable to change, as we go through formal internal naming process. No updates just yet / for this PR; I expect to be ready with those updates before the RC (April 13).